### PR TITLE
fix readme yolov8

### PR DESCRIPTION
**Pull request by chengtianwu**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/layout/yolov8/README.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ Please [download](#2-results) the exported MindIR file first, or refer to the [M
 python tools/export.py --model_name_or_config configs/layout/yolov8/yolov8n.yaml --data_shape 800 800 --local_ckpt_path /path/to/local_ckpt.ckpt
 ```
 
-The `data_shape` is the model input shape of height and width for MindIR file. The shape value of MindIR in the download link can be found in [Notes](#2-results) under results table.
+The `data_shape` is the model input shape of height and width for MindIR file. The shape value of MindIR in the download link can be found in [Notes](#2-results) under results table. `distribute` in yaml must be set to False.
```
Comment (panshaowu) [inline configs/layout/yolov8/README.md:143]: 建议"must be"->"shall be"

**Comment by panshaowu** [comment]
请在本PR中，刷新首页“What is new”和“Model list”章节，目前暂未体现版面检测内容。

File: configs/layout/yolov8/README.md (hunk 1 above)
Comment (chengtianwu) [reply configs/layout/yolov8/README.md]: 已修改

**Comment by chengtianwu** [comment]
> 请在本PR中，刷新首页“What is new”和“Model list”章节，目前暂未体现版面检测内容。

**Comment by chengtianwu** [comment]
> 请在本PR中，刷新首页“What is new”和“Model list”章节，目前暂未体现版面检测内容。

已修改

**Comment by chengtianwu** [comment]
重新打开

File: README_CN.md [hunk 2]
```diff
@@ -170,6 +170,12 @@ python tools/eval.py \
 - [x] [ABINet](configs/rec/abinet/README_CN.md) (CVPR'2021)
 </details>
 
+<details open markdown>
+<summary>版面分析</summary>
+
+- [x] [YOLOv8](configs/layout/yolov8/README_CN.md) (ultralytics)
```
Comment (panshaowu) [inline README_CN.md:176]: ultralytics链接到对方公司网页？

Comment (chengtianwu) [reply README_CN.md]: 已修改

**Review by panshaowu — APPROVED** [review]

**Review by tonytonglt — APPROVED** [review]

**Pull request merged**